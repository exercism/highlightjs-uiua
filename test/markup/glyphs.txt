◠ ⌵ + ⌝ ’ ⍤ ∠ ˜ ◡ ⋯ ∩ □ ⊓ ⊸ ⍩ ⌈ ⑄ ⊛ ℂ ◇ ⊟ ◴ ∂ ♭ ⊙ ÷ ⍢ ↘ . ∵ = ⍖ ⬚ ⌕ ⊢ ¤ : ⌊ ∧ ⊃ ⋅ ≥ &gt; ⊕ ∘ ⊗ ∫ ⍚ ⊂ ▽ ⊣ ⧻ ≤ &lt; ₙ ⦷ ≍ ↥ ∊ ∈ ↧ ◿ × ¯ ¬ ≠ ⌅ ⤚ ⟜ ⤸  ⋕ ⊜ ⊡ ◌ ⁿ ⚂ ⇡ / ⍥ ☇ ↯ ⇌ ⍏ ↻ ⁅ ≡ \\ ⊏ △ ± ∿ ⍆ √ ? - ⨬ ⊞ ↙ ⸮ ⍉ ◹ ⍣ ⧅ ° ⍜ ◰ ⊚ ◫ ⤙