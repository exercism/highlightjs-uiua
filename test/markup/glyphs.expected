<span class="hljs-built_in">◠</span> <span class="hljs-built_in">⌵</span> <span class="hljs-built_in">+</span> <span class="hljs-built_in">⌝</span> <span class="hljs-built_in">’</span> <span class="hljs-built_in">⍤</span> <span class="hljs-built_in">∠</span> <span class="hljs-built_in">˜</span> <span class="hljs-built_in">◡</span> <span class="hljs-built_in">⋯</span> <span class="hljs-built_in">∩</span> <span class="hljs-built_in">□</span> <span class="hljs-built_in">⊓</span> <span class="hljs-built_in">⊸</span> <span class="hljs-built_in">⍩</span> <span class="hljs-built_in">⌈</span> <span class="hljs-built_in">⑄</span> <span class="hljs-built_in">⊛</span> <span class="hljs-built_in">ℂ</span> <span class="hljs-built_in">◇</span> <span class="hljs-built_in">⊟</span> <span class="hljs-built_in">◴</span> <span class="hljs-built_in">∂</span> <span class="hljs-built_in">♭</span> <span class="hljs-built_in">⊙</span> <span class="hljs-built_in">÷</span> <span class="hljs-built_in">⍢</span> <span class="hljs-built_in">↘</span> <span class="hljs-built_in">.</span> <span class="hljs-built_in">∵</span> <span class="hljs-built_in">=</span> <span class="hljs-built_in">⍖</span> <span class="hljs-built_in">⬚</span> <span class="hljs-built_in">⌕</span> <span class="hljs-built_in">⊢</span> <span class="hljs-built_in">¤</span> <span class="hljs-built_in">:</span> <span class="hljs-built_in">⌊</span> <span class="hljs-built_in">∧</span> <span class="hljs-built_in">⊃</span> <span class="hljs-built_in">⋅</span> <span class="hljs-built_in">≥</span> &amp;<span class="hljs-built_in">g</span><span class="hljs-built_in">t</span>; <span class="hljs-built_in">⊕</span> <span class="hljs-built_in">∘</span> <span class="hljs-built_in">⊗</span> <span class="hljs-built_in">∫</span> <span class="hljs-built_in">⍚</span> <span class="hljs-built_in">⊂</span> <span class="hljs-built_in">▽</span> <span class="hljs-built_in">⊣</span> <span class="hljs-built_in">⧻</span> <span class="hljs-built_in">≤</span> &amp;<span class="hljs-built_in">l</span><span class="hljs-built_in">t</span>; <span class="hljs-built_in">ₙ</span> <span class="hljs-built_in">⦷</span> <span class="hljs-built_in">≍</span> <span class="hljs-built_in">↥</span> <span class="hljs-built_in">∊</span> <span class="hljs-built_in">∈</span> <span class="hljs-built_in">↧</span> <span class="hljs-built_in">◿</span> <span class="hljs-built_in">×</span> <span class="hljs-built_in">¯</span> <span class="hljs-built_in">¬</span> <span class="hljs-built_in">≠</span> <span class="hljs-built_in">⌅</span> <span class="hljs-built_in">⤚</span> <span class="hljs-built_in">⟜</span> <span class="hljs-built_in">⤸</span>  <span class="hljs-built_in">⋕</span> <span class="hljs-built_in">⊜</span> <span class="hljs-built_in">⊡</span> <span class="hljs-built_in">◌</span> <span class="hljs-built_in">ⁿ</span> <span class="hljs-built_in">⚂</span> <span class="hljs-built_in">⇡</span> <span class="hljs-built_in">/</span> <span class="hljs-built_in">⍥</span> <span class="hljs-built_in">☇</span> <span class="hljs-built_in">↯</span> <span class="hljs-built_in">⇌</span> <span class="hljs-built_in">⍏</span> <span class="hljs-built_in">↻</span> <span class="hljs-built_in">⁅</span> <span class="hljs-built_in">≡</span> <span class="hljs-built_in">\</span><span class="hljs-built_in">\</span> <span class="hljs-built_in">⊏</span> <span class="hljs-built_in">△</span> <span class="hljs-built_in">±</span> <span class="hljs-built_in">∿</span> <span class="hljs-built_in">⍆</span> <span class="hljs-built_in">√</span> <span class="hljs-built_in">?</span> - <span class="hljs-built_in">⨬</span> <span class="hljs-built_in">⊞</span> <span class="hljs-built_in">↙</span> <span class="hljs-built_in">⸮</span> <span class="hljs-built_in">⍉</span> <span class="hljs-built_in">◹</span> <span class="hljs-built_in">⍣</span> <span class="hljs-built_in">⧅</span> <span class="hljs-built_in">°</span> <span class="hljs-built_in">⍜</span> <span class="hljs-built_in">◰</span> <span class="hljs-built_in">⊚</span> <span class="hljs-built_in">◫</span> <span class="hljs-built_in">⤙</span>